Cargo.toml

```diff
@@ -1,10 +1,5 @@
 [workspace]
-members = [
-  "bin/pcl", 
-  "crates/phoundry",
-  "crates/common",
-  "crates/da",
-]
+members = ["bin/pcl", "crates/phoundry", "crates/common", "crates/da"]
```

Comment:
You seem to have forgotten to run `cargo fmt`. 

As a best practice always run `cargo clippy` and `cargo fmt` before opening a PR

Reply:
hmm. funny, I did and it made this into one line.